packages/mongodb-memory-server-core/src/__tests__/singleDB-test.ts

```diff
@@ -1,33 +1,33 @@
-import { Db, MongoClient } from 'mongodb';
-import MongoMemoryServer, { MongoInstanceDataT } from '../MongoMemoryServer';
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { MongoClient } from 'mongodb';
+import MongoMemoryServer from '../MongoMemoryServer';
 
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 600000;
-let con: MongoClient;
-let db: Db;
-let mongoServer: MongoMemoryServer;
 
-beforeAll(async () => {
-  mongoServer = new MongoMemoryServer();
-  const mongoUri = await mongoServer.getUri();
-  con = await MongoClient.connect(mongoUri, {
-    useNewUrlParser: true,
-    useUnifiedTopology: true,
-  });
+describe('Single mongoServer', () => {
+  let con: MongoClient;
+  let mongoServer: MongoMemoryServer;
 
-  db = con.db(await mongoServer.getDbName());
-});
+  beforeAll(async () => {
+    mongoServer = await MongoMemoryServer.create();
+    con = await MongoClient.connect(mongoServer.getUri(), {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    });
+  });
 
-afterAll(async () => {
-  if (con) {
-    con.close();
-  }
-  if (mongoServer) {
-    await mongoServer.stop();
-  }
-});
+  afterAll(async () => {
+    if (con) {
+      await con.close();
+    }
+    if (mongoServer) {
+      await mongoServer.stop();
+    }
+  });
 
-describe('Single mongoServer', () => {
   it('should start mongo server', async () => {
+    const db = con.db(mongoServer.instanceInfo!.dbName);
```

Comment:
The exclamation mark is evil 😈 Honestly, I don't remember why exactly I'm avoiding it in my code. But let it keep in the test suites.

Reply:
i mean i know and understand why, but in the tests *you literally expect* the value to be defined (and not null / undefined)

Reply:
Yep, I understand this. My comment was just a note. Feel free to merge this PR.

Reply:
yes i know, i will rebase this pr then merge it (when im at it again)